lms/djangoapps/mobile_api/groups/views.py

```diff
@@ -249,104 +64,93 @@ class GroupsInvite(generics.CreateAPIView):
-                            {   "name": "Marc Ashman", 
-                                "id": "10154833899435243"},
-                            {   "name": "Peter Organa", 
-                                "id": "10154805420820176"},
-                            {   "name": "Joey Freund", 
-                                "id": "1279985874"}, 
-                            {   "name": "Yin Zhuoqun", 
-                                "id": "1600206076"},    
-                            {   "name": "David Liu", 
-                                "id": "1658520223"},
-                            {   "name": "Andrew Joe", 
-                                "id": "120401174"}, 
-                            {   "name": "Gaelan D'costa", 
-                                "id": "122600141"}, 
-                            {   "name": "Hafiz Vellani", 
-                                "id": "122609084"}, 
-                            {   "name": "Adir Krafman", 
-                                "id": "500301193"}, 
-                            {   "name": "Naeem Lakhani", 
-                                "id": "502193756"}, 
-                            {   "name": "Alex Mann", 
-                                "id": "502576321"}, 
-                            {   "name": "Natasha Dalal", 
-                                "id": "506753913"}, 
-                            {   "name": "Adam Borzecki", 
-                                "id": "507174319"}, 
-                            {   "name": "Bryant Balatbat", 
-                                "id": "512414329"}, 
-                            {   "name": "Nahim Nasser", 
-                                "id": "516528519"}, 
-                            {   "name": "Farzana Nasser", 
-                                "id": "572710051"}, 
-                            {   "name": "Aaron Ritchie", 
-                                "id": "578710450"}, 
-                            {   "name": "Karthik Ramakrishnan", 
-                                "id": "584467742"}, 
-                            {   "name": "Sinai Gross", 
-                                "id": "591867536"}, 
-                            {   "name": "Ani Tumanyan", 
-                                "id": "617573112"}, 
-                            {   "name": "Paul Jeffrey Crowe", 
-                                "id": "635195067"}, 
-                            {   "name": "Eitan Cohen", 
-                                "id": "657887138"}, 
-                            {   "name": "Eli Atlas", 
-                                "id": "674674636"}, 
-                            {   "name": "Mark Reale", 
-                                "id": "676480219"}]
-                })
+    def delete(self, request, *args, **kwargs):
+        graph = facebook.GraphAPI(facebook.get_app_access_token(_APP_ID, _APP_SECRET))
+        post_args = {'method' : 'delete'}
+        url = _FACEBOOK_API_VERSION + _APP_ID + "/groups/" + kwargs['group_id']
```

Comment:
i'm always weary of using things like kwargs['group_id']. 

The scenario that has stung me in the past is when group_id is not in kwargs. This could be guaranteed if you're using a rigid well defined serializer --- but if you're not, then i would make sure to handle that scenario so a catastrophic 500 error isn't thrown.

You could handle this through a try/catch or by using kwargs.get('group_id', None) which returns None if group_id isn't there. Then you can handle the scenario with an if statement.

Both approaches are valid.